extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -322,4 +344,15 @@ public function getSearchSuggestions() {
 			$this->sendRequest( 'SearchSuggestionsApi', 'getList', $this->request->getParams() )->getData()
 		);
 	}
+
+	private function getMainPageData() {
+		$mainPageData = [];
+		try {
+			$curatedContentData = $this->sendRequest( 'CuratedContent', 'getList' )->getData();
+			$mainPageData['curatedContent'] = $this->mercuryApi->getCuratedContent( $curatedContentData );
+		} catch ( NotFoundApiException $ex ) {
+			WikiaLogger::instance()->error( 'Curated content and categories are empty' );
```

Comment:
true. If curated content is empty then Mercury won't display it. No need for drama.